src/components/TablePeople/TablePeople.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+
+export const TablePeople: React.FC = () => {
+  const [getpeople, setGetPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```